[2016-05-19 11:57:03,938: WARNING/MainProcess] /Users/Herbee/anaconda/lib/python2.7/site-packages/celery/apps/worker.py:161: CDeprecationWarning: 
Starting from version 3.2 Celery will refuse to accept pickle by default.

The pickle serializer is a security concern as it may give attackers
the ability to execute any command.  It's important to secure
your broker from unauthorized access when using pickle, so we think
that enabling pickle should require a deliberate action and not be
the default choice.

If you depend on pickle then you should set a setting to disable this
warning and to be sure that everything will continue working
when you upgrade to Celery 3.2::

    CELERY_ACCEPT_CONTENT = ['pickle', 'json', 'msgpack', 'yaml']

You must only enable the serializers that you will actually use.


  warnings.warn(CDeprecationWarning(W_PICKLE_DEPRECATED))
[2016-05-19 11:57:04,094: INFO/MainProcess] Connected to redis://localhost:6379//
[2016-05-19 11:57:04,107: INFO/MainProcess] mingle: searching for neighbors
[2016-05-19 11:57:05,120: INFO/MainProcess] mingle: all alone
[2016-05-19 11:57:05,137: WARNING/MainProcess] w1@Herb.local ready.
[2016-05-19 11:57:23,438: INFO/MainProcess] Received task: tasks.count[d51caadb-ff90-4d0d-a351-5bcc00c8e1c2]
[2016-05-19 11:57:23,441: WARNING/Worker-1] 5
[2016-05-19 11:57:24,443: WARNING/Worker-1] 6
[2016-05-19 11:57:25,444: WARNING/Worker-1] 7
[2016-05-19 11:57:26,446: WARNING/Worker-1] 8
[2016-05-19 11:57:27,429: INFO/MainProcess] Terminating d51caadb-ff90-4d0d-a351-5bcc00c8e1c2 (15)
[2016-05-19 11:57:27,455: ERROR/MainProcess] Task tasks.count[d51caadb-ff90-4d0d-a351-5bcc00c8e1c2] raised unexpected: Terminated(15,)
Traceback (most recent call last):
  File "/Users/Herbee/anaconda/lib/python2.7/site-packages/billiard/pool.py", line 1678, in _set_terminated
    raise Terminated(-(signum or 0))
Terminated: 15
